kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -58,20 +58,16 @@
       downloadMediaText() {
         return this.$tr('downloadMedia');
       },
+      downloadLink() {
+        const filePath = this.availableFiles[0].storage_url;
+        // Extract filename from the storage url.
+        const filename = filePath.replace(/^.*[\\\/]/, '');
+        const newFilename = this.sanitizeFilename(this.title);
+        return `/downloadcontent/${filename}/${newFilename}`;
```

Comment:
This URL should be constructed using `Kolibri.urls` or probably, here, `this.kolibri.urls` (see other examples in the code) rather than being hardcoded.